src/rustdoc-json-types/lib.rs

```diff
@@ -54,8 +54,8 @@ pub struct ItemSummary {
     ///
     /// Note that items can appear in multiple paths, and the one chosen is implementation
     /// defined. Currently, this is the full path to where the item was defined. Eg
-    /// [`String`] is currently `["alloc", "string", "String"]` and [`HashMap`] is
-    /// `["std", "collections", "hash", "map", "HashMap"]`, but this is subject to change.
+    /// [`String`] is currently `["alloc", "string", "String"]` and [`FxHashMap`] is
+    /// `["rustc_hash", "FxHashMap"]`, but this is subject to change.
```

Comment:
@aDotInTheVoid  I am not sure if I should mention about `FxHashMap` since it's not a standard rust type. Let me know if I should exclude it. Please re-queue the PR otherwise.

Reply:
Yeah, this should be `HashMap`, as this is making a point that the path given reference internals that don't exist in the public API. I think you can do something like ```[`HashMap`][`std::collections::HashMap`]```